pom.xml

```diff
@@ -314,6 +314,11 @@
                 <version>${fabric8.kubernetes-client.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>io.fabric8</groupId>
+                <artifactId>openshift-model-operatorhub</artifactId>
+                <version>${fabric8.openshift-client.version}</version>
+            </dependency>
```

Comment:
Should it be test scope dependency?

Reply:
It's not directly in `test` scope, so I can't add it there.